flink-flux-core/src/main/java/com/uber/athena/flux/flink/compiler/api/CompilerVertex.java

```diff
@@ -34,7 +34,7 @@
   private int compiledSourceCount;
   private DataStream dataStream = null;
```

Comment:
This should not be contained in this. 

`compilerVertex` should be abstract and needs to have an impl specific to the datastream API components.